examples/run_consisid.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -x
+
+export PYTHONPATH=$PWD:$PYTHONPATH
+
+# CogVideoX configuration
+SCRIPT="consisid_example.py"
+MODEL_ID="/storage/hxy/cvpr2025/weight/ConsisID-preview"
+INFERENCE_STEP=50
+
+mkdir -p ./results
+
+# CogVideoX specific task args
+TASK_ARGS="--height 480 --width 720 --num_frames 49"
+
+# CogVideoX parallel configuration
+N_GPUS=6
+PARALLEL_ARGS="--ulysses_degree 2 --ring_degree 3"
+# CFG_ARGS="--use_cfg_parallel"
+
+# Uncomment and modify these as needed
+# PIPEFUSION_ARGS="--num_pipeline_patch 8"
+# OUTPUT_ARGS="--output_type latent"
+# PARALLLEL_VAE="--use_parallel_vae"
+ENABLE_TILING="--enable_tiling"
+# COMPILE_FLAG="--use_torch_compile"
+INPUT_IMAGE='--image_path /storage/hxy/cvpr2025/ConsisID/asserts/example_images/4.png'
```

Comment:
Could you please use a relative path here and give instruction to users how to download this image?